### ISSUE TITLE
Why not collect "Unerrored Codewords"?

### ISSUE THREAD

**Issue by jkasten2** [sarabveer/arris_cable_modem_stats#3]
### Summary
Why not collect "Unerrored" in-addition to "Correctable" and "Uncorrectable" Codewords counts the python script already collects?

### Motivation
The "Unerrored" count would be good to have to compare between "Correctable" and "Uncorrectable" so failure percentages can be calculated.
![image](https://github.com/user-attachments/assets/9147ff54-4cec-47a1-abe6-3ede68c6feef)

### Availability 
![image](https://github.com/user-attachments/assets/4662e542-05b5-46c3-9067-74d985cdb1a9)
"Unerrored Codewords" is available on this device at least:
```
Product Type: XB8
HW Version: 2.0
Download Version: Prod_22.2_d31 & Prod_22.2
Model: CGM4981COM
```

##### XB8 Collection
The follow changes can be made by adding the following lines above each noted location:

Add `channel['unerrored'] = cm_error_rows[1].find_all("td")[i].text.strip()` above:
https://github.com/sarabveer/cable-modem-stats/blob/503b378c15682b0a842fa439fcd5fb6e44f14aef/src/comcast_xb8_stats.py#L111

##### Send to Influx
Add `'unerrored': int(stats_down['unerrored']),` above:
https://github.com/sarabveer/cable-modem-stats/blob/503b378c15682b0a842fa439fcd5fb6e44f14aef/src/__main__.py#L207

I tested these two lines on an XB8, however I am not sure what happens if `unerrored` doesn't exist for other modems. `'unerrored': int(stats_down['unerrored']),`  might throw if `unerrored` is missing, so there might need to be a check before reading the value.

### PR
I am happy to put up a PR to add this, however any guidance on handling missing `unerrored` in `__main__.py` when it isn't support for the modem?

**Comment by sarabveer** [comment]
Hi, thanks for opening this issue.

Initially I used this with Arris modems which do not give Unerrored codewords in the webui.

**Comment by jkasten2** [comment]
@sarabveer I see, thanks for the background!

I believe I can account for some modems not having the value, is this a feature you would consider merging in if I make a PR?

**Comment by sarabveer** [comment]
Yea a PR is fine, I will look into handling the case if unerrored is empty.

**Comment by jkasten2** [comment]
@sarabveer I created a PR for this and accounted for other modems not having the field in PR https://github.com/sarabveer/cable-modem-stats/pull/5
* I had to make some other changes, so you probably should review https://github.com/sarabveer/cable-modem-stats/pull/4 first

**Comment by sarabveer** [comment]
I tested the PRs locally and all looked OK, they have been merged.

Closed by #4 #5 #6
